tools/validate_spec/validation_tool.py

```diff
@@ -0,0 +1,305 @@
+import argparse
```

Comment:
The tool is supposed to be named `tools/validate_spec.py`.

Reply:
For command line use a shebang line pointing to the python interpreter should be present (when executing on my Debian system it hangs without that):

```sh
#!/usr/bin/env python3
```

Reply:
The script should be executable (mode 755).

Reply:
Fixed in f057148.

Reply:
The changes go into the right direction. I have two issues, though:

1. Generic Python (like `KeyError`) are used, but we just realized that this is a bad idea in pyrflx as you cannot distinguish real bugs from usage errors. Please create a tool-specific exception and use it in the error case
2. As exceptions are raised, user errors look like the program has just crashed (with a call backtrace, etc.). Please catch the newly introduced exception in the main program and just print a message (and set the exit code of the program to non-zero).

Reply:
Refactored error handling in b6d55eb, 2efbb4b

Reply:
Error handling has been changed in multiple commits.

> Generic Python (like KeyError) are used, but we just realized that this is a bad idea in pyrflx as you cannot distinguish real bugs from usage errors. Please create a tool-specific exception and use it in the error case

This has been changed by d75b93a